.github/CODEOWNERS

```diff
@@ -36,3 +36,6 @@
 /pkg/ui/embedded.go
 /pkg/ui/src/js/protos.d.ts
 /pkg/ui/src/js/protos.js
+
+/docs/generated/http         @cockroachdb/http-api-prs
+/pkg/cmd/docgen/http.go
```

Comment:
I don't recall - do you need to repeat the target on every line or is the first auto-propagated to every line after that?

Reply:
Thanks for this tip, I amended this change to be explicit on each line.